### DISCUSSION TITLE
Clarifications on mappings (TC5 beads in aromatic rings)

### DISCUSSION THREAD

**Discussion by bo-pang** [ricalessandri/Martini3-small-molecules#22] [Q&A] [answered]
When chlorobenzene is mapped into a CG model, it is represented as (TC5)2-SX3. The SX3 represents the chlorine atom, the carbon atom on the benzene ring attached to the chlorine, and the two carbon atoms connected to this carbon atom(also are shared with neighboring beads). I'm not sure which carbon atoms on the benzene ring do the other two TC5 represent? Do the other two TC5 represent two carbons and three carbons, respectively, or do they both represent three carbons, including the carbon atom on the opposite position to the chlorine in the benzene ring and are included in the both TC5?

**Comment by ricalessandri** [comment, marked as answer]
Hi @bo-pang , I uploaded a sketch that may be helpful: https://github.com/ricalessandri/Martini3-mappings/blob/main/chlorobenzene.pdf
The SX3 bead (in green in the sketch) represents the chlorine, the carbon atom immediately connected to it (let's call it "alpha carbon"), and 1/2 of each of the two atoms connected to the alpha carbon (let's call these two "beta carbons"). Hence, SX3 maps 3 non-hydrogen atoms: `1*Cl + 1*C_alpha + 2*(1/2 * C_beta)`. This still leaves one with `2*(1/2 * C_beta)` and the 3 remaining carbon atoms to be mapped, i.e., still 4 carbon atoms to be mapped. Each of the other TC5 beads indeed maps to 2 C atoms, as I try to show in https://github.com/ricalessandri/Martini3-mappings/blob/main/chlorobenzene.pdf. Only that each of them maps `1/2*C_beta + 1*C_gamma + 1/2*C_delta` (assuming that C_gamma is the carbon in meta position with respect to the chlorine and C_delta is the carbon in para position). 
Let me know if this clears (or not) your doubts!

**Comment by Clare-xiong** [comment]
Thanks for your replying, @bo-pang is my friend. I understand how to map the chlorobenzene. But for the pyrrole, the TN6d bead (in blue in the sketch) represents the nitrogen, the carbon atom immediately connected to it ("alpha carbon"), and 1/2 of each of the two atoms connected to the alpha carbon (two "beta carbons"). Do the rest two TC5 beads represent the four 1/2 of carbon? I try to show it in the picture.
<img src="https://user-images.githubusercontent.com/85223463/229534801-92413f63-fe2c-4943-bf62-8828657d351d.jpg" height="400">

**Comment by ricalessandri** [comment]
Hi @Clare-xiong , great that chlorobenzene is clear.

Regarding pyrrole, I uploaded the mapping explanation here: https://github.com/ricalessandri/Martini3-mappings/blob/main/pyrrole.pdf. You are right about the TN6d bead. In principle, we'd be left with 3 carbons (`2 x 1/2 C_beta and 2 x C_gamma`). However, we still use 2 more TC5 beads to keep the aromatic ring structure (i.e., you need 3 beads to make a ring). Hence, we "overmap" the structure a bit, i.e., we use a bit more beads than what would be needed if you just counted the number of non-hydrogen atoms (1 x T-bead = 2 x non-H atoms). So, eventually, the TC5 beads are taken two describe 2 x C atoms each (the C_beta and the C_gamma), with the C_beta being "overmapped". Let me know if this clears (or not) your doubts!

**Comment by Clare-xiong** [comment]
Thank you very much for your answer. It's clear for me.
